chipsec/hal/mmio.py

```diff
@@ -22,32 +22,32 @@
 Access to MMIO (Memory Mapped IO) BARs and Memory-Mapped PCI Configuration Space (MMCFG)
 
 usage:
-    >>> read_MMIO_reg(cs, bar_base, 0x0, 4)
-    >>> write_MMIO_reg(cs, bar_base, 0x0, 0xFFFFFFFF, 4)
-    >>> read_MMIO(cs, bar_base, 0x1000)
-    >>> dump_MMIO(cs, bar_base, 0x1000)
+    >>> read_MMIO_reg(bar_base, 0x0, 4)
+    >>> write_MMIO_reg(bar_base, 0x0, 0xFFFFFFFF, 4)
+    >>> read_MMIO(bar_base, 0x1000)
+    >>> dump_MMIO(bar_base, 0x1000)
 
     Access MMIO by BAR name:
 
-    >>> read_MMIO_BAR_reg(cs, 'MCHBAR', 0x0, 4)
-    >>> write_MMIO_BAR_reg(cs, 'MCHBAR', 0x0, 0xFFFFFFFF, 4)
-    >>> get_MMIO_BAR_base_address(cs, 'MCHBAR')
-    >>> is_MMIO_BAR_enabled(cs, 'MCHBAR')
-    >>> is_MMIO_BAR_programmed(cs, 'MCHBAR')
-    >>> dump_MMIO_BAR(cs, 'MCHBAR')
-    >>> list_MMIO_BARs(cs)
+    >>> read_MMIO_BAR_reg('MCHBAR', 0x0, 4)
+    >>> write_MMIO_BAR_reg('MCHBAR', 0x0, 0xFFFFFFFF, 4)
+    >>> get_MMIO_BAR_base_address('MCHBAR')
+    >>> is_MMIO_BAR_enabled('MCHBAR')
+    >>> is_MMIO_BAR_programmed('MCHBAR')
+    >>> dump_MMIO_BAR('MCHBAR')
+    >>> list_MMIO_BARs()
 
     Access Memory Mapped Config Space:
 
-    >>> get_MMCFG_base_address(cs)
-    >>> read_mmcfg_reg(cs, 0, 0, 0, 0x10, 4)
-    >>> read_mmcfg_reg(cs, 0, 0, 0, 0x10, 4, 0xFFFFFFFF)
+    >>> get_MMCFG_base_address()
+    >>> read_mmcfg_reg(0, 0, 0, 0x10, 4)
+    >>> read_mmcfg_reg(0, 0, 0, 0x10, 4, 0xFFFFFFFF)
 """
 from typing import List, Optional, Tuple
 from chipsec.hal import hal_base
 from chipsec.library.exceptions import CSReadError
 from chipsec.library.logger import logger
-from chipsec.library.defines import get_bits
+from chipsec.library.defines import get_bits, is_all_ones
```

Comment:
Is this supposed to be **is_field_all_ones** to match line 248?

Reply:
I don't believe so, is_field_all_ones is on line 215 and it's being called on the register object, not imported.